src/PageContent/Settings/AccountManager.js

```diff
@@ -0,0 +1,191 @@
+                                </div>
+                                :
+                                <span />
+                            }
+
+                            {this.props.accountType.accountType === AccountType.DONATING_AGENCY_MEMBER && !this.props.accountType.isAdmin ?
+                                <span />
+                                :
+                                <div className="amd-edit amd-edit-1">
+                                    <button type="button" className="form-button confirm-button" onClick={this.handleEdit.bind(this)}>Edit</button>
+                                </div>
+                            }
+
+                        </div>
+
+                        :
+
+                        <form onSubmit={this.handleSubmit.bind(this)}>
+                            
+                            <div className="editing-box">
+                                <div className="editing-child-1">
+                                    <label className="label-component details">Name</label><br /><br />
+                                    <label className="label-component details">Title/Position</label><br /><br />
+                                    <label className="label-component details">Email</label><br /><br />
+                                    <label className="label-component details">Phone Number</label><br /><br />
+                                </div>
+                                
+                                <div className="editing-child-2 amd-editing-2">
+                                    <input name="name" type="text" className="form-input" defaultValue={this.props.account.name} /><br /><br />
+                                    <input name="position" type="text" className="form-input" defaultValue={this.props.account.position} /><br /><br />
+                                    <input name="email" type="email" className="form-input" defaultValue={this.props.account.email} /><br /><br />
+                                    <input name="phone" type="tel" pattern={StringFormat.PHONE} className="form-input" defaultValue={this.props.account.phone} /><br /><br />
+                                </div> 
+
+                                <div className="editing-child-3">
+                                    Recieve Notifications via<br />
+                                    <h6>Feature coming soon!</h6>
+                                    {/* <input type="checkbox" name="smsNotif" value="smsNotif" defaultChecked={this.props.account.smsNotif}/><label className="label-component details">SMS/Text Message</label><br />
+                                    <input type="checkbox" name="emailNotif" value="emailNotif" defaultChecked={this.props.account.emailNotif}/><label className="label-component details">Email</label> */}
+                                </div>    
+                            </div>
+
+                            {this.props.secondaryContact ?
```

Comment:
`this.props.account.accountType === AccountType.RECEIVING_AGENCY ?`
we should allow them to add one if they didn't have one